proto/storage/vulnerability.proto

```diff
@@ -51,6 +51,8 @@ message EmbeddedVulnerability {
 
   VulnerabilitySeverity severity = 19; // @gotags: policy:"Severity"
   VulnerabilityState state = 20; // @gotags: search:"Vulnerability State"
+  repeated CVSSScore cvss_metrics = 21;
```

Comment:
Suggestion: Rename to cvss _info, you can check with PM/UX.(metrics has a different connotation that I am not sure applies here?)

Reply:
+1

Reply:
scanner is adding this field cvss_metris to list of each unique CVSS metric based on the source

Reply:
https://github.com/stackrox/stackrox/blob/master/scanner/decisions/0010-nvd-cvss-scores.md